release: v1.53.0 (#5873)
apollographql/router: .changesets/maint_bnjjj_improve_perf_custom_telemetry.md

Reviewer: @bnjjj please review this edited changeset

> ### Improve performance, don't re-create meter and instruments on every calls in Telemetry ([PR #5629](https://github.com/apollographql/router/pull/5629))

Reviewer: ```suggestion
### Improve performance by optimizing telemetry meter and instrument creation ([PR #5629](https://github.com/apollographql/router/pull/5629))
```

> The creation of otel instruments using a regex is no longer part of the hot path. Now we create these instruments when starting the telemetry plugin and not in every serives.

Reviewer: ```suggestion
The router's performance has been improved by removing telemetry creation out of the critical path, from being created in every service to being created when starting the telemetry plugin.
```
